runtime/src/main/scala/akka/grpc/javadsl/ServiceHandler.scala

```diff
@@ -50,11 +49,27 @@ object ServiceHandler {
   @varargs
   def handler(handlers: JFunction[HttpRequest, CompletionStage[HttpResponse]]*)
       : JFunction[HttpRequest, CompletionStage[HttpResponse]] = {
-    val servicesHandler = concat(handlers: _*)
+    val servicesHandler = _concat(handlers: _*)
     (req: HttpRequest) => if (sServiceHandler.isGrpcRequest(req)) servicesHandler(req) else unsupportedMediaType
   }
 
-  private[javadsl] def concat(handlers: JFunction[HttpRequest, CompletionStage[HttpResponse]]*)
+  @varargs
+  def concat(handlers: akka.japi.function.Function[HttpRequest, CompletionStage[HttpResponse]]*)
```

Comment:
I don't think we need this one anymore, we should just be able to use the `concatOrNotFound`

Reply:
Ok. will do

Reply:
done